api/queries_repo.go

```diff
@@ -1220,3 +1220,39 @@ func CreateRepoTransformToV4(apiClient *Client, hostname string, method string,
 		IsPrivate: responsev3.Private,
 	}, nil
 }
+
+// MapReposToIDs retrieves a set of IDs for the given set of repositories.
+// This is similar logic to RepoNetwork, but only fetches databaseId and does not
+// discover parent repositories.
+func MapReposToIDs(client *Client, host string, repositories []ghrepo.Interface) ([]int64, error) {
```

Comment:
This name is not great, perhaps we should change it to `GetRepoIDs`.

Reply:
renamed.